packages/tqdm/test_tqdm.py

```diff
@@ -0,0 +1,14 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["tqdm"], pytest_assert_rewrites=False)
+def test_tqdm(selenium):
+    import time
+    import warnings
+
+    import tqdm
+
+    with warnings.catch_warnings(record=True) as w:
+        for _ in tqdm(range(100), desc="Processing"):
+            time.sleep(0.1)
```

Comment:
Could you reduce the number of interations or the sleep time? We don't want to spend 10 seconds for testing this simple feature.